src/xcode/ENA/ENA/Source/SceneDelegate.swift

```diff
@@ -118,15 +118,32 @@ final class SceneDelegate: UIResponder, UIWindowSceneDelegate, RequiresAppDepend
 
 	func userNotificationCenter(_: UNUserNotificationCenter, didReceive response: UNNotificationResponse, withCompletionHandler completionHandler: @escaping () -> Void) {
 		switch response.notification.request.identifier {
-		case ActionableNotificationIdentifier.testResult.identifier,
-			 ActionableNotificationIdentifier.riskDetection.identifier,
+		case ActionableNotificationIdentifier.riskDetection.identifier,
 			 ActionableNotificationIdentifier.deviceTimeCheck.identifier:
 			showHome(animated: true)
 
 		case ActionableNotificationIdentifier.warnOthersReminder1.identifier,
 			 ActionableNotificationIdentifier.warnOthersReminder2.identifier:
-			showPositiveTestResultFromNotification(animated: true)
+			checkIfPositiveTestResultWasShown()
 
+		case ActionableNotificationIdentifier.testResult.identifier:
+			guard let testResultRawValue = response.notification.request.content.userInfo[ActionableNotificationIdentifier.testResult.identifier] as? Int,
```

Comment:
Wow, `response.notification.request.content.userInfo[ActionableNotificationIdentifier.testResult.identifier]` is just…
![6a3b1480-d93d-11e9-82c8-3780f102a035](https://user-images.githubusercontent.com/149976/102365934-61aa1180-3fb8-11eb-8560-367092179193.png)

Reply:
I used two screens for that :'D
Will try to make it more readable